fv/ipip_test.go

```diff
@@ -327,6 +327,14 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ IPIP topology before adding
 		BeforeEach(func() {
 			ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
 			defer cancel()
+			if bpfEnabled {
+				infra.RemoveNodeAddresses(felixes[0])
```

Comment:
Why this difference?

Reply:
In case of bpf, we send felixes[0].name in the list options. My understanding is that list nodes will return the node object corresponding the felixes[0] and we disable BGP in felixes[0]. https://github.com/projectcalico/felix/blob/master/fv/ipip_test.go#L331